src/firebase/firebase.example.ts

```diff
@@ -1,26 +0,0 @@
-// Import the functions you need from the SDKs you need
```

Comment:
Why have you deleted this file? Keep it as it is.

Reply:
i have not deleted this file just changed the name

Reply:
That's deletion lmao.
You had to make a copy of it and rename the copy, not the source.